backend/service/k8s/pods.go

```diff
@@ -326,10 +326,30 @@ func getPodStatus(pod *corev1.Pod) string {
 		reason = pod.Status.Reason
 	}
 
+	totalInitContainers := len(pod.Spec.InitContainers)
+	nativeSidecarRestartPolicy := corev1.ContainerRestartPolicyAlways
+	nativeSidecarContainerId := map[int]struct{}{}
+
+	for i := range pod.Spec.InitContainers {
+		if pod.Spec.InitContainers[i].RestartPolicy != nil && pod.Spec.InitContainers[i].RestartPolicy == &nativeSidecarRestartPolicy {
+			// if the init container has a restart policy, it is native sidecar and should not be counted
+			// https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/753-sidecar-containers/README.md
+			totalInitContainers--
+			nativeSidecarContainerId[i] = struct{}{}
+		}
+	}
+
 	initializing := false
+	initContainerCount := 0
 	for i := range pod.Status.InitContainerStatuses {
```

Comment:
One goal of showing initializing containers can be - show customers that their application container run is waiting on initializing of platform containers.

So i would think it would be really nice to include the sidecar containers in this information, since now they are all in initContainers.

* normal init container -- make sure it's terminated and exit code is 0
* sidecar init container -- make sure it's running and ready